### PR TITLE
Add Dependabot config

### PR DESCRIPTION
**Pull request by tgeoghegan**
Configure dependabot to update Rust crates and GitHub Actions weekly.

### CODE REVIEW

**Review by jbr — APPROVED** [review]

**Comment by jbr** [comment]
Do I not have merge permissions?

**Review by divergentdave — APPROVED** [review]

**Comment by tgeoghegan** [comment]
> Do I not have merge permissions?

That's odd. You have "Write" permissions on the repository which says it lets you "manage issues and pull requests", which I would think would include merging them. Maybe we have to add you to CODEOWNERS?

**Comment by jbr** [comment]
Seems so, the other two PRs opened by dependabot are blocked merge by me: `divviup/committers
divviup/committers was requested for review as a code owner`

**Pull request merged**
